pkgs/applications/networking/syncthing/default.nix

```diff
@@ -25,15 +25,28 @@ stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p $out/bin
+    mkdir -p $out/bin $out/etc/systemd/{system,user}
+
     cp bin/* $out/bin
+
+    substitute etc/linux-systemd/system/syncthing-resume.service \
+               $out/etc/systemd/system/syncthing-resume.service \
+               --replace /usr/bin/pkill ${procps}/bin/pkill
```

Comment:
Absolutely - those files are only relevant on Linux anyway. I'll sort it out later. Thanks for the feedback @Mic92